app/src/main/java/com/conupods/Settings/DefaultPreferences.java

```diff
@@ -0,0 +1,20 @@
+package com.conupods.Settings;
+
+import android.content.SharedPreferences;
+
+import com.conupods.R;
+
+public class DefaultPreferences {
+
+    public static void setDefaultPreferences(SharedPreferences preferences) {
```

Comment:
these are the default preferences for the Settings page, so function should be renamed to setDefaultPreferencesForSettingsPage . Also instead of making this static, make it into a regular class, and turn shared preferences into a member variable